lib/msal-common/src/utils/Constants.ts

```diff
@@ -271,6 +272,7 @@ export enum CacheType {
  */
 export const APP_METADATA = "appmetadata";
 export const ClientInfo = "client_info";
+export const THE_FAMILY_ID = "1";
```

Comment:
nit: wondering if theres a reason why it's THE_FAMILY_ID, as opposed to just FAMILIY_ID?

Reply:
As we have only one family id. :-D and nothing is going to change unless we break a ton of apps out there. As far as I know, they want to get out of FOCI model than add more families.